src/components/tracked-controls.js

```diff
@@ -210,8 +220,8 @@ module.exports.Component = registerComponent('tracked-controls', {
    * @returns {boolean} Whether button has changed in any way.
    */
   handleButton: function (id, buttonState) {
-    var changed = this.handlePress(id, buttonState) ||
-                  this.handleTouch(id, buttonState) ||
+    var changed = this.handlePress(id, buttonState) |
```

Comment:
This is the core bug fix. We no longer do the short circuit operator. Instead we process all three handle* calls so that all events get generated and prior button states get saved correctly.